src/middleware.js

```diff
@@ -118,7 +118,14 @@ function apiMiddleware({ getState }) {
     }
 
     // We can now dispatch the request FSA
-    next(await actionWith(requestType, [action, getState()]));
+    if (
+      typeof requestType.payload === 'function' ||
+      typeof requestType.meta === 'function'
```

Comment:
How about the other `requestType` properties that can be passed as functions? ie, `headers`, `options` or `bailout`

Reply:
`headers`, `options` or `bailout` aren't properties of `requestType`, they are properties of `[RSAA]`, thus this change doesn't apply to them.